distribution/ddf-common/src/main/resources/etc/org.ops4j.pax.url.mvn.cfg

```diff
@@ -75,7 +79,10 @@ org.ops4j.pax.url.mvn.disableAether=true
 #
 # The following property value will add the system folder as a repo.
 #
-org.ops4j.pax.url.mvn.defaultRepositories=file:${karaf.home}/${karaf.default.repository}@snapshots@id=karaf.${karaf.default.repository}
+org.ops4j.pax.url.mvn.defaultRepositories=\
+    file:${karaf.home}/${karaf.default.repository}@id=system.repository@snapshots, \
+    file:${karaf.data}/kar@id=kar.repository@multi@snapshots, \
```

Comment:
❓ What does the `@multi` option do? Couldn't find any reference to it on the OPS4J Pax URL wiki (https://ops4j1.jira.com/wiki/display/paxurl/Mvn+Protocol).